steam_app/controllers/GamesViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  ViewController.swift
+//  steam_app
+//
+//  Created by Khumoyiddin on 28/12/22.
+//
+
+import UIKit
+
+class GamesViewController: UIViewController {
+    
+    @IBOutlet weak var tableView: UITableView!
+
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let nib = UINib(nibName: "GamesTableViewCell", bundle: nil)
```

Comment:
viewDidLoad should not be flooded. It would be better to take all related tableView configuration lines to a separate method like -configureTableView()